src/cloud/howtos/install-components.md

```diff
@@ -66,116 +37,135 @@ To update `composer.json`:
    composer require pixlee/magento2:1.0.1 --no-update
    ```
 
+1. Update the project dependencies.
+
    ```bash
    composer update
    ```
 
-1. Wait for project dependencies to update.
-1. Enter the following commands in the order shown to commit your changes, including `composer.lock`:
+1. Add, commit, and push code changes.
 
    ```bash
    git add -A
    ```
 
    ```bash
-   git commit -m "<message>"
+   git commit -m "Install <extension-name>"
    ```
 
    ```bash
-   git push magento <environment ID>
+   git push origin <branch-names>
    ```
 
-If there are errors, see [extension deployment failure]({{ site.baseurl }}/cloud/trouble/trouble_comp-deploy-fail.html).
+   {:.bs-callout-warning}
+   When installing an extension, you must include the `composer.lock` file when you push code changes to the remote environment. The `composer install` command reads the `composer.lock` file to enable the defined dependencies in the remote environment.
 
-{:.bs-callout-warning}
-When installing and adding the module, you must add the `composer.lock` to your Git branch for deployment. If the extension is not in the file, the extension won't load in {{site.data.var.ece}}. This ensures when the `composer install` command is used, the extension properly loads. This command uses the `composer.lock` file.
+1. After the build and deploy finishes, use a SSH to log in to the remote environment and verify the extension installed.
 
-### Step 3: Verify the extension {#verify}
+   ```bash
+   bin/magento module:status <extension-name>
+   ```
 
-To verify the extension installed properly, you can check its functionality in the Magento Admin or you can view enabled modules using the CLI:
+   An extension name uses the format: `<VendorName>_<ComponentName>`.
 
-1. Open a terminal.
-1. [Checkout the branch]({{ site.baseurl }}/cloud/before/before-setup-env-2_clone.html#branch) where the module is installed.
-1. List all enabled modules:
+   Sample response:
 
-   ```bash
-   php bin/magento module:status
+   ```terminal
+   Module is enabled
    ```
 
-1. Verify the extension is listed.
+   If you encounter deployment errors, see [extension deployment failure][trouble].
 
-The extension name is in the format `<VendorName>_<ComponentName>`. It will not be in the same format as the Composer name.
+## Manage extensions
 
-## Manage extensions {#manage}
+When you add an extension using Composer, the deployment process automatically enables the extension. If you already have the extension installed, you can enable or disable the extension using the CLI. When managing extensions, use the format: `<VendorName>_<ComponentName>`. Never enable or disable an extension while logged in to the remote environments.
```

Comment:
My extension was not enabled, and I wasn't able to enable it manually.  Maybe because I have not purchased the extension.

Reply:
I am thinking there may be one thing I missed. Possibly the .magento.app.yaml extensions sections. I will look into this.

Reply:
I would check your `config.php` file. You should have: `'Magento_B2b' => 1,`
And of course, your `composer.lock` file should have: 
```
        {
            "name": "magento/extension-b2b",
            "version": "x.x.x",
       ...
```

Reply:
Unless, you weren't talking about b2b...

Reply:
I cleared the cache on the server, and now the B2B module status is as expected